ui/app/routes/workspace/projects/project/app/deployment.ts

```diff
@@ -38,8 +38,10 @@ export default class DeploymentDetail extends Route {
   }
 
   async model(params: DeploymentModelParams): Promise<Deployment.AsObject> {
+    let { deployments } = this.modelFor('workspace.projects.project.app');
+    let { id: deployment_id } = deployments.find((obj) => obj.sequence == params.sequence);
```

Comment:
[sand] Same threequals comment:

```suggestion
    let { id: deployment_id } = deployments.find((obj) => obj.sequence === params.sequence);
```